### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by killy10o10**
# Project Description
Use forms to add new `Post`, `Comment`, and `Like` for the Rails Blog app.

## Feature Description
- Created a method called `#current_user` in the ApplicationController to make current user data available to all controllers.
`#current_user` returns the first user from the database.
> Created forms to perform the following functions:
- Creates a new `Post` on behalf of the `current_user`.
- Creates a `Comment` on behalf of the `current_user`.
- Allow User (`current_user`) to `Like` a `Post`.

Screen Shot 2022-12-17 at 14 54 39

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -7,4 +7,32 @@ def show
...
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+
+    render :new, locals: { post: @post }
+  end
+
+  def create
```
Comment (Munsa1) [inline app/controllers/posts_controller.rb:17]: - [x] Great work on your project, however when I click on the create post button, I get the following syntax error. kindly fix this.

Before button click
image

After
image

### Note that this error occurs when I try to click on the `comments` and `likes` links

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @killy10o10, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (killy10o10) [reply app/controllers/posts_controller.rb:17]: That is strange. I do not have that error on my end 

https://user-images.githubusercontent.com/89058561/208250302-58944b33-f42a-41f7-941c-2fd19d709b92.mp4

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @killy10o10 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**